### PR TITLE
feature/CU-jva2fz_BACK-END-Create-quiz-service

### PR DESCRIPTION
**Pull request by YesLife**

### CODE REVIEW

**Comment by YesLife** [comment]
Task linked: [CU-jva2fz [BACK-END] Create quiz service](https://app.clickup.com/t/jva2fz)

**Comment by Thurstag** [comment]
Task linked: [CU-jva2fz [BACK-END] Create quiz service](https://app.clickup.com/t/jva2fz)

**Comment by Thurstag** [comment]
Il faut initialiser le service quiz dans le container de services

File: src/index.js [hunk 1]
```diff
@@ -52,6 +53,9 @@ server.listen(serverConfig.port, async () => {
         await database.init();
         LOGGER.info("[Main] Connected successfully to the MongoDB database");
 
+        // Quiz service
+        quizService = new QuizService(database);
```
Comment (Thurstag) [inline src/index.js]: dans le container

**Review by Thurstag — CHANGES_REQUESTED** [review]

**Review by Thurstag — APPROVED** [review]

**Review by Tomansion — APPROVED** [review]

**Review by furtivesock — APPROVED** [review]

**Pull request merged**